src/styles/components/get-involved/mobilepopupCard.module.scss

```diff
@@ -0,0 +1,48 @@
+/*.multicontentcard {
+    display: flex;
+    align-items: center;
+    font-family: var(--font-inter);
+    border-radius: 12px;
+    background-color: var(--off-white, #FFFBF0);
+    //height: 100vh;
+}
+*/
+
+.layout {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: var(--font-inter);
+    border-radius: 12px;
+    background-color: var(--off-white, #FFFBF0);
+    width: 19.625rem;
+    height: 36.8125rem;
+
+    h1 {
+        color: var(--maroon, #5B1110);
+        text-align: center;
+        font-family: Jost;
+        font-size: 25px;
+        width: 15.3125rem;
+        height: 1.875rem;
+        flex-shrink: 0;
+    }
+
+    p {
+        font-family: "Be Vietnam";
+        font-size: 15px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: 170%;
+        text-align: center;
+        width: 13.6875rem;
+        height: 27.625rem;
+        flex-shrink: 0;
+    }
+
+    .button {
+        color: var(--maroon, #5B1110);
```

Comment:
the styles aren't showing up because the `.button` class is inside the `.layout` selector but the button isn't nested in this div, so just move this outside of the bottom brace and you should start seeing the effect